Eatery/Controllers/EateriesViewController.swift

```diff
@@ -34,6 +33,7 @@ class EateriesViewController: UIViewController, MenuButtonsDelegate, CLLocationM
     }()
     
     fileprivate var userLocation: CLLocation?
+    fileprivate let dateFormatter = DateFormatter()
```

Comment:
In my opinion, DateFormatter.dateFormat should be closely associated with the date formatter. 
Use something like: 

`fileprivate let dateFormatter = {
let dateFormatter = DateFormatter()
dateFormatter.dateFormat = "YYYY-MM-dd"
return dateFormatter
}()
`

Reply:
Another thought would be to extend DateFormatter and provide a convenience initializer on it with the format.